<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1584</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Install new mod</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Choose destination</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_settings">
         <property name="text">
          <string>Create backup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="initialize_mod">
         <property name="text">
          <string>LETS GO MAH DUDES</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="num_mods"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableWidget" name="mod_list">
            <property name="contextMenuPolicy">
             <enum>Qt::NoContextMenu</enum>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::NoDragDrop</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Folder</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Subfolder</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Enabled</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="move_up">
               <property name="text">
                <string>UP</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="move_down">
               <property name="text">
                <string>DOWN</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="file_view"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1584</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInstall_new_mod">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionInstall_new_mod"/>
   </widget>
   <addaction name="menuInstall_new_mod"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInstall_new_mod">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Install new mod</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
